### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by orenbenkiki** [tanaylab/metacells#58]
This issue will be updated every time we create a new release or have other important updates that impact metacells users. If you are using the python metacells package, you are encouraged to clock on the "subscribe" button to the right, and githib will update you every time we post an update here.

**Comment by orenbenkiki** [comment]
**New release 0.9.3**

*TL;DR:* Users are encouraged to `pip install --upgrade metacells` to obtain this newer version as this should produce better-sized metacells by default. If needed, it also allows for better manual control over the metacell sizes.

*Compatibility:*

This API of this version is backward compatible with 0.9.[0-2], and is extended to allow specifying a new parameter `target_metacell_umis` (default of 160,000) in addition to the existing `target_metacell_size` in cells (whose default reduced from 96 to 48).

The metacells computed by this version will be different from these computed by versions 0.9.[0-2].

*Details:*

This version solves the issue that previous 0.9.[0-2] versions metacell sizes were too sensitive to the number of UMIs in the cells, which created too-large metacells in most cases.

It is always advisable to look at the distribution of UMIs per metacell and cells per metacell (in the QC page in MCView) and consider adjusting one of the targets to better fit your data.

**Comment by orenbenkiki** [comment]
Published version 0.9.5 with bug fixes. The major one is a avoiding a potential deadlock due to `threadpoolctl` not being thread safe (which you'd expect with the word "thread" _right there in the name_ ... sigh), plus some other very minor fixes. Results "should" be identical to 0.9.4 and 0.9.3 (given you use the same version of numpy and scipy).
